pb/service.proto

```diff
@@ -6,6 +6,14 @@ option go_package = ".;pb";
 service Callhandling {
   rpc Ping (PingRequest)            returns (PingResponse);
   rpc CreateCall(CreateCallRequest) returns (CallhandlingResponse) {}
+
+  rpc DialEvent(EventRequest) returns (CallhandlingResponse) {}
```

Comment:
We should probably talk through this - from my understanding each event should be in the past tense and might not share the same request message... but without enough context the code itself looks ok

Reply:
👌 messaging now to schedule discussion